src/components/Header.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+function Header() {
+  const [isLogin, setIsLogin] = useState(false);
```

Comment:
u can refer [this comment](https://github.com/2-guys-Javascript/travel-web-app/pull/25#discussion_r1367158110)